R/rsad.R

```diff
@@ -23,6 +52,17 @@ rsad <- function(S, frac, sad, trunc=NaN, Pois.samp=TRUE, k, zeroes=FALSE, ssize
   return(y)
 }
 
+rfixed <- function(com, frac, ssize) {
+    rad.tab <- rad(ssize*com)
```

Comment:
I did not grasp why multiply the abudances in the community by the sample size, see my comment on this function in the pull request.